### DISCUSSION TITLE
Reproduce evaluation results from paper

### DISCUSSION THREAD

**Discussion by gjj0430** [RealVNF/deep-rl-network-service-coordination#8] [General] [answered]
Dear author ，it's me ，I have asked you my confusion about DeepCoord .  Because I didn't find your e-mail , so I have to open an issue about my questions .  In the paper ’Self-Driving Network and Service Coordination Using Deep Reinforcement Learning‘ , when you did the numerical evaluation compared to the baseline , the data you have displayed is reproduced by yourselves or obtained from their works? I wan to reproduce the evaluation and I hope you could help me. Thank you. And my e-mail is **gjj0430bupt@gmail.com**. Looking forward to your reply. @stefanbschneider

**Comment by qarawlus** [comment, marked as answer]
Hello, we have reproduced the baseline data ourselves. The shortest path (SP) and load balancing (LB) algorithms can be found here https://github.com/RealVNF/baseline-algorithms. We have additionally developed an adapter to allow the BSP heuristic to function with our coord-sim simulator. This adapter can be found here https://github.com/RealVNF/bjointsp-adapter. 

@stefanbschneider I propose that you add a discussion section to the repository so that we can keep the issues section for more technical problems.

**Reply by stefanbschneider** [reply]
BSP itself is also open source: https://github.com/CN-UPB/B-JointSP

So all algorithms that we compared against are public, similarly our adjusted version of BSP ("BSP Ad." in the paper) as @qarawlus mentioned.

I hope this helps? You'll still have to set up and configure all evaluation parameters (network, scenario, ...), depending on which experiments you want to perform.

**Reply by gjj0430** [reply]
Thank you, sir , it could help me a lot.
